templates/pupil.jade

```diff
@@ -92,10 +92,34 @@ block main
 
               include pupil-svg
 
+
+
     //- full width background image
     div.Grid
       div(class="Background-img--full-wh-grid Background-img-cover" style="background-image: url('../media/images/home/slides/01.jpg');")
 
+    section(style="background-color:#6f7679;")
+      div.Site-content-container
+        a(name="pupil-mobile")
+        div.Feature-container-column
+        div.Grid.Grid--1of2.Grid--justifyCenter.Grid--center.Grid--alignContentCenter.Feature-row
+          div.Grid-cell
+            div(style="color: white;").Feature
+              h1 Pupil Mobile
+              p Pupil Mobile enables you to connect your Pupil eye tracking headset to your Android device via USBC.
+              p Preview video and other sensor streams on your Android device. Record video and other sensor streams locally on your Android device or stream data over your WiFi network to other computers (clients) running Pupil Capture.
+              p Pupil Mobile is seamlessly integrated with Pupil Capture. You can use Pupil Capture to remotely monitor and control instances of Pupil Mobile.
+              p Pupil Mobile is currently in public beta. Download it on the Google Play Store.
```

Comment:
Google Play Store is a link to https://play.google.com/store/apps/details?id=com.pupillabs.pupilmobile&hl=en